### ISSUE TITLE
Delay Profiles don't work as advertised

### ISSUE THREAD

**Issue by Taloth** [Sonarr/Sonarr#578]
In a discussion with @markus101 we concluded there are several scenarios where the delay profiles don't work as we hoped.

Particularly when considering usenet vs torrent delays we get scenarios that simply don't make sense.

**Comment by markus101** [comment]
@Taloth do you remember what the cases were? If not lets close this one.

**Comment by Taloth** [comment]
Yes, back then it didn't delay properly, it delayed everything but didn't grab the best release after expiry, also some trickery with cutoff iirc.
I really should write a couple of usecases and test em.

**Comment by markus101** [comment]
Work was done around delay profiles (though mostly around delayed items not being cleaned up).

If we don't have any specific issues right now, let's close this out an re-open or open a new one. Right now there in't anything actionable here.

**Comment by Taloth** [comment]
I'd like to keep this ghi as a reminder to get evaluate the behavior. But I'll change it to a task.
